ROMFS/px4fmu_common/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+############################################################################
+#
+#   Copyright (c) 2016 PX4 Development Team. All rights reserved.
```

Comment:
Wrong copyright year.

Reply:
I have updated the Copyright header with a 2018 header.

I also added a 2018 copyright header to the ROMFS/CMakeLists.txt file - it previously didn't have one at all. Should the one for the ROMFS/CMakeLists.txt be a purely 2018 copyright header, or should we make it a 20XX - 2018 header?

Reply:
2018 only is fine for the new ones.